File: Python/Legacy/Path.py
#! /usr/bin/env python3
# -*- coding: utf-8 -*-

import os
from Utility import *

def Path():

	pt=os.path.normpath(os.path.join(os.getcwd(),__file__))

	prints(f"""
		ホームディレクトリ:       {os.path.expanduser("~")}
		カレントディレクトリ:     {os.getcwd()}

		このファイルのフルパス:   {os.path.abspath(pt)}
		このファイルの実行パス:   {__file__}
		このファイルの名前:       {os.path.basename(pt)}
		このファイルのある場所:   {os.path.dirname(pt)}
	""")

	pnl(2)

runningDirectly(__name__,Path)

File: JavaScript/Legacy/Path.js
#! /usr/bin/env node

let process = require("process");
let os = require("os");
let u=require("./Utility.js");
let e=exports;

e.Path=()=>{

	u.println(`
		ホームディレクトリ:       ${os.homedir()}
		カレントディレクトリ:     ${process.cwd()}

		このファイルのある場所:   ${__dirname}
	`);

	u.pnl(2);

};

if (!module.parent) e.Path();